src/providers/base_javascript.js

```diff
@@ -198,14 +190,25 @@ export default class Base_javascript {
 		}
 		const args = this._updateLockFileCmdArgs(manifestDir);
 		try {
-			invokeCommand(this._cmdName(), args)
+			this.#invokeCommand(args);
 		} catch (error) {
-			throw new Error(`failed to create lockfile "${args}" - Error: ${error}`, {cause: error});
+			throw new Error(`failed to create lockfile "${args}"`, { cause: error });
```

Comment:
I see you removed the `try catch` from other places, if youre keeping this one because of the `finally`, note you can have `try finally` without the `catch` :slightly_smiling_face:

Reply:
In this case I found useful the chained error messages but maybe it's not that useful.